client/src/pages/Medical.js

```diff
@@ -67,9 +67,15 @@ const Medical = (props) => {
   };
 
   const fetchGeoData = () => {
-    const geolocation = getGeoLocation();
+    // Defaulting to San Francisco
+    // TODO fix geolocation
```

Comment:
ah ok so I guess you're just testing here if it works with the geo API, doesn't it work anymore with HTML5 geo coords?

Reply:
Yeah so I kept getting an empty geolocation, so I defaulted it to SF lat/long to test that the call to the backend works. I also removed the authentication check on the geo endpoint for this reason, and I don't believe that it should be there since unauthenticated users can get to this page.

Reply:
I guess I should really try using a different location than the city I'm in just in case it was just a fluke that it didn't work... 🤔

Reply:
depends on the type of error response? was the API just completley unreachable or did you get an error response because of the input?

Reply:
The api call to our geoservice endpoint failed when the geolocation object was empty, since passing in the latitude and  longitude query parameters was required. I don't know why the geolocation object was empty. So I think the HTML5 geolocation failed, so we should fix that in another ticket.

Reply:
The api call to our geoservice endpoint failed when the geolocation object was empty, since passing in the latitude and  longitude query parameters was required. I don't know why the geolocation object was empty. So I think the HTML5 geolocation failed, so we should fix that in another ticket.

Reply:
Ah ok, first version that I made it was working so probably regression, I think it was only making the request when the user had given permission to get the location with html5 geo api